src/tool.rs

```diff
@@ -441,14 +455,24 @@ impl Tool {
 ///
 /// Detection of a family is done on best-effort basis and may not accurately reflect the tool.
 #[derive(Copy, Clone, Debug, PartialEq)]
+#[non_exhaustive]
 pub enum ToolFamily {
     /// Tool is GNU Compiler Collection-like.
+    #[non_exhaustive]
```

Comment:
I don't think you need to put this ahead of each variant.

Reply:
We do, otherwise _fields on_ variants aren't `non_exhaustive`.

Reply:
Oh yeah...thanks